### PR TITLE
DataType.Null

### PR DESCRIPTION
**Pull request by TaroEld**
Adds ::MSU.Utils.DataType.Null to deSer.

### CODE REVIEW

**Review by Enduriel — APPROVED** [review]
When merging this in I'd squash the two null commits first but other than that all good from my side.

**Comment by LordMidas** [comment]
Adding the Null data type should be a `feat` commit whereas changing the `::MSU.serialize` to `this.serialize` should be a separate `refactor` commit.

File: scripts/config/msu/utils.nut [hunk 1]
```diff
@@ -60,12 +61,16 @@
 
 				case "array":
 					_out.writeU8(this.DataType.Array);
-					::MSU.serialize(element, _out);
+					this.serialize(element, _out);
```
Comment (LordMidas) [inline scripts/config/msu/utils.nut:64]: Split the changing of `::MSU.` to `this` into a separate `refactor` commit to keep it separate from the `feat` of adding the Null data type.

**Review by LordMidas — CHANGES_REQUESTED** [review]

File: scripts/config/msu/utils.nut (hunk 1 above)
Comment (TaroEld) [reply scripts/config/msu/utils.nut:64]: I'll just put that as a fix on dev

**Comment by TaroEld** [comment]
This will get merged via other means

**Pull request closed without merging**